LMR/documents/Companies.json

```diff
@@ -1,7 +1,7 @@
 {
-   "path": "/organizations/LMR/projects/devel/collections/Companies/documents",
-   "method": "POST",
-   "content": [
+	"path": "/organizations/LMR/projects/devel/collections/cmp/documents/",
+	"method": "POST",
+	"content": [
```

Comment:
[Programmers who use spaces in their code get paid more than those who use tabs](http://www.businessinsider.com/programmers-who-use-spaces-get-paid-more-than-those-who-use-tabs-2017-6)